ucx/.clang-format

```diff
@@ -21,9 +21,9 @@ AllowShortLoopsOnASingleLine: false
 AllowShortEnumsOnASingleLine: false
 AllowDesignatedInitializersOnASingleLine: false
 AlwaysBreakAfterReturnType: None
-PenaltyReturnTypeOnItsOwnLine: 5
-PenaltyBreakAssignment: 200
-PenaltyExcessCharacter: 200
+PenaltyReturnTypeOnItsOwnLine: 100
+PenaltyBreakAssignment: 100
+PenaltyExcessCharacter: 10
```

Comment:
low penalty here means more likely to keep lines longer, IMO it should be higher

Reply:
Fixed